### ISSUE TITLE
build integration test failure

### ISSUE THREAD

**Issue by colinsurprenant** [elastic/logstash#6949]
As reported in #6930

```
Failures:
  1) Test Monitoring API can retrieve JVM stats
     Failure/Error: expect(result["jvm"]["uptime_in_millis"]).to be > 100
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # ./specs/monitoring_api_spec.rb:46:in `(root)'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:79:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:95:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:91:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:123:in `try'
     # ./specs/monitoring_api_spec.rb:44:in `(root)'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
Finished in 14 minutes 23 seconds (files took 6.41 seconds to load)
34 examples, 1 failure
Failed examples:
rspec ./specs/monitoring_api_spec.rb:39 # Test Monitoring API can retrieve JVM stats
Randomized with seed 14328
```
```
Failures:
  1) Test Monitoring API can retrieve queue stats
     Failure/Error: expect(result["pipeline"]["queue"]).not_to be_nil
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # ./specs/monitoring_api_spec.rb:57:in `(root)'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:79:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:95:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:91:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:123:in `try'
     # ./specs/monitoring_api_spec.rb:55:in `(root)'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
  2) Test Monitoring API can retrieve JVM stats
     Failure/Error: expect(result["jvm"]["uptime_in_millis"]).to be > 100
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # ./specs/monitoring_api_spec.rb:46:in `(root)'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:79:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:95:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:91:in `try'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/stud-0.0.22/lib/stud/try.rb:123:in `try'
     # ./specs/monitoring_api_spec.rb:44:in `(root)'
     # /home/travis/.rvm/gems/jruby-1.7.25/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
Finished in 12 minutes 55 seconds (files took 5.37 seconds to load)
34 examples, 2 failures
Failed examples:
rspec ./specs/monitoring_api_spec.rb:50 # Test Monitoring API can retrieve queue stats
rspec ./specs/monitoring_api_spec.rb:39 # Test Monitoring API can retrieve JVM stats
Randomized with seed 63523
```

**Comment by colinsurprenant** [comment]
same error in #6944

**Comment by jsvd** [comment]
fixed by #7074
